checker/run

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -ex
+
+. $(dirname "$0")/../options
```

Comment:
Yep, i have it integrated with flycheck. I've addressed an additional complaint about double quoting, there are two remaining, `Can't follow non constant source`, which would be catched by set -e if the options file is not there, and the usage of the variables defined in the options file before defined, which is covered by the `[ -z "$var" ]` checks. Pls let me know if you can spot any further  problem.

Thanks! :)